Misc/NEWS.d/next/Core and Builtins/2019-07-21-11-11-58.bpo-37616.0pEhuj.rst

```diff
@@ -0,0 +1 @@
+Update `getpath` detect zip path in :mod:`sys.path`
```

Comment:
Please use proper sentences in news entries.

I believe using single backquotes around "getpath" is incorrect (as per https://devguide.python.org/documenting). Also, I think the full file path should be used (instead of just "getpath").

`sys.path` is not a module.